examples/gallery/images/grdlandmask.py

```diff
@@ -0,0 +1,25 @@
+"""
+Create 'wet-dry' mask grid
+--------------------------
+The :meth:`pygmt.grdlandmask` method allows setting 
+all nodes on land or water to a specified value using
+the ``maskvalues`` parameter. 
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+
+# Define region of interest
+region = [-65, -40, -40, -20]
+
+# Assign a value of 0 for all water masses and a value of 1 for all land masses
+grid = pygmt.grdlandmask(region=region, spacing="5m", maskvalues=[0, 1], resolution="l")
+
+# Plot clipped grid
+fig.basemap(region=region, projection="M12c", frame=True)
+
+fig.grdimage(grid=grid, cmap="lajolla")
+fig.colorbar(position="JMR+o0.5c/0c+w8c")
```

Comment:
I think it makes more sense to have a discrete CPT file, with 0 and 1 representing water and land, respectively.

You can refer to the `makecpt` call at this [example](https://www.pygmt.org/v0.4.1/gallery/symbols/points_categorical.html).

Reply:
Done!